spring-rest-http/src/main/java/com/baeldung/jsonparam/jsonParamOpenApiSwaggerDefinition

```diff
@@ -0,0 +1,154 @@
+swagger: '2.0'
+...
```

Comment:
Are we putting the ... here and other places below as a shorthand notation to say that in a real spec, there would be more here? That's fine for snippets in the article, but the code in the repository should reflect complete and syntactically correct examples. In fact, the article has 2.0 and 3.0 examples, so we should have one file here for the 2.0 spec and one for the 3.0 spec. Both should be valid specs.